tokio-threadpool/src/worker/mod.rs

```diff
@@ -79,7 +79,7 @@ struct CurrentTask {
 ///
 /// This identifier is unique scoped by the thread pool. It is possible that
 /// different thread pool instances share worker identifier values.
-#[derive(Debug, Clone, Hash, Eq, PartialEq)]
+#[derive(Debug, Clone, Copy, Hash, Eq, PartialEq)]
```

Comment:
Given that the type is already `Clone`, I would avoid `Copy` for now (forwards compat hazard).